packages/renderer/src/lib/kube/details/KubePort.svelte

```diff
@@ -75,6 +78,19 @@ async function removePortForward(): Promise<void> {
 <span aria-label="port {port.value}" class="flex gap-x-2 items-center">
   {port.displayValue}
   {#if mapping}
+
+
+  {#if localhostAddress}
+
+  <CopyToClipboard
+  clipboardData={localhostAddress}
+  title={localhostAddress}
+  />
+  
+  {/if}
```

Comment:
suggestion: I think something goes wrong on the formatting/indenting (yes biomejs is fast but do not format html part)